### ISSUE TITLE
Completion only partially working for plugins

### ISSUE THREAD

**Issue by FooSoft** [folke/neodev.nvim#82]
Hi there, thanks for the great plugin :)

Just started using this the other day, but it looks like I'm just getting the raw sumneko output for plugins (no neodev.nvim docs, certain things missing). Everything works perfectly inside of `init.lua` and all the other scripts that it includes directly.

I'm initializing sumneko as follows:

```lua
    nvim_lspconfig.sumneko_lua.setup({
        settings = {
            Lua = {
                diagnostics = {globals = {'vim'}},
                runtime = {version = 'LuaJIT'},
                telemetry = {enable = false},
                workspace = {library = vim.api.nvim_get_runtime_file('', true)},
            },
        },
    })
```

I'm *not* explicitly initializing neodev.nvim, so it should be running with defaults. I have tried to call setup as well, providing a list of the plugins I wish to debug, but it made no difference.

I'm using the built-in Vim plugin structure and am not using a package manager. My directory structure looks like this:

```
.
├── ftplugin
│   ├── c.lua
│   ├── cpp.lua
│   ├── go.lua
│   ├── html.lua
│   ├── markdown.lua
│   └── yaml.lua
├── init.lua
├── install.bat
├── install.sh
├── link.bat
├── link.sh
├── lua
│   ├── autocmd.lua
│   ├── common.lua
│   ├── ginit.lua
│   ├── lsp.lua
│   ├── plugins.lua
│   └── util.lua
├── pack
│   └── plugins
│       └── start
│           ├── hflip.nvim
│           │   ├── LICENSE
│           │   ├── lua
│           │   │   └── hflip.lua
│           │   └── plugin
│           │       └── hflip.lua
│           ├── neodev.nvim
...
```

So basically, if I'm editing something like `init.lua` or `plugins.lua`, neodev.nvim works. If I'm editing `hflip.lua` it doesn't work. Any idea what I might be messing up here?

`LspInfo` output looks the same to me for working and non-working cases:

![Screenshot from 2022-10-22 21-11-51](https://user-images.githubusercontent.com/544478/197373355-08a8c20d-b6a7-44b5-aecb-6a36dbfa3bad.png)
![Screenshot from 2022-10-22 21-11-36](https://user-images.githubusercontent.com/544478/197373360-196cbdf7-57bd-4496-b13c-a6ea40f78994.png)

**Comment by FooSoft** [comment]
Playing around with this some more, it appears that I can get down to the `start` directory and still have neodev.nvim function correctly. As soon as I move into `hflip.nvim` directory, it stops.

**Comment by FooSoft** [comment]
Figured it out. I needed to call `setup` even if I am using defaults. That was the problem.
